ccvm_simulators/solution.py

```diff
@@ -18,6 +18,9 @@ class Solution:
         solve_time (float): Time to solve the problem.
         pp_time (float):  Time to post-process the problem.
         optimal_value (float): The optimal objective value for the given problem instance.
+        best_value (float): The best objective value for the given problem instance.
+        num_frac_values (int): Number of fractional values in the solutions of given problem instance.
+        optional_info (list): Optional information in a given problem instance.
```

Comment:
Similar change here. Describe as the solution to the problem instance instead of optional information.

Reply:
done